Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -174,11 +169,22 @@ public class IonSerializationOptions
         public readonly bool PermissiveMode;
         
         public Dictionary<Type, IIonSerializer> IonSerializers { get; init; }
+        public Dictionary<string, object> CustomContext { get; init; }
+    }
+
+    public interface IIonSerializerFactory
+    {
+        IIonSerializer Create(IonSerializationOptions options, Dictionary<string, object> customContext);

```

Comment:
Similarly, for public API methods, can you please add docs?

Reply:
Added.

Reply:
Can you also add for the Create method itself?

Reply:
Added.

Reply:
Thanks, good start. Can you also add the <params> and <returns> tags for the input parameters and the return value?

Reply:
Added more docs.